app/views/unattended/provisioning_templates/snippet/preseed_kernel_options_autoinstall.erb

```diff
@@ -30,7 +35,14 @@ description: options for the kernel / preseed startup initialization
     options << "ip=[#{iface.ip6}]::[#{subnet6.gateway}]:[#{subnet6.mask}]:#{hostname}:#{iface.identifier}:none:[#{subnet6.dns_servers.join(']:[')}]"
   end
 
+  options << 'ramdisk_size=1500000'
+  options << 'fsck.mode=skip'
+  options << 'autoinstall'
+  options << "url=http://#{@preseed_server}#{image_path}"
+  options << 'cloud-config-url=/dev/null'
+  options << "ds=nocloud-net;s=http://#{foreman_request_addr}/userdata/#{mac ? mac + '/' : ''}"
   options << 'console-setup/ask_detect=false'
+  options << "locale=#{host_param('lang') || 'en_US'}"
   options << 'localechooser/translation/warn-light=true'
   options << 'localechooser/translation/warn-severe=true'
   options << "hostname=#{hostname}"
```

Comment:
Do you think you can maybe adjust the order of parameters here such that the snapshots of PXEGrub2 and PXELinux don't change at all?

Reply:
Definitely... if I could have arranged it better to reduce the churn in the snapshots then that's the way to go.

Reply:
Given the 2 existing snippets I can't see a way to arrange this any better.